src/encoded/schemas/gene.json

```diff
@@ -0,0 +1,73 @@
+{
+    "title": "Gene",
+    "description": "Schema for submitting a gene.",
+    "id": "/profiles/gene.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["organism", "symbol"],
+    "identifyingProperties": ["uuid", "symbol-organism.name"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/standard_status" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
```

Comment:
I guess we are going to add something like coordinates: { 'GRCh38': { chromosome: "X", start: "44949", end: "65990"}, "hg19": {P...} for version 2?

Reply:
I'd say v2 since I'm not sure how this will be used. Here is my current thought on schema:
```
"coordinates": {
    "title": "Coordinates",
    "description": "Genome positions for different assemblies of this gene.",
    "type": "array",
    "default": [],
    "uniqueItems": true,
    "items": {
        "title": "Coordinate",
        "description": "Genome positions for one assembly of this gene.",
        "type":  "object",
        "additionalProperties": false,
        "required":["assembly", "chromosome", "start", "end"],
        "properties": {
            "assembly": {
                "title": "Mapping assembly",
                "description": "The genome assembly to which modification coordinates relate. e.g. GRCh38.",
                "type": "string",
                "enum": [
                    "ce10",
                    "ce11",
                    "dm3",
                    "dm6",
                    "GRCh38",
                    "hg19",
                    "mm10",
                    "mm9"
                    ]
            },
            "chromosome": {
                "title": "Chromosome",
                "description": "The number (or letter) designation for the target chromosome. e.g. 1, 2L, or X",
                "type": "string"
            },
            "start": {
                "title": "Start",
                "description": "The starting coordinate.",
                "type": "integer"
            },
            "end": {
                "title": "End",
                "description": "The ending coordinate (possibly equal to the starting coordinate for one base.)",
                "type": "integer"
            }
        }
    }
}
```